ivre/tools/ipinfo.py

```diff
@@ -127,12 +127,14 @@ def disp_recs_std(flt):
 
 def disp_recs_short(flt):
     for addr in db.passive.distinct('addr', flt=flt):
-        print(ivre.utils.int2ip(addr))
+        if addr is not None:
+            print(ivre.utils.int2ip(addr))
 
 
 def disp_recs_distinct(field, flt):
     for value in db.passive.distinct(field, flt=flt):
-        print(value)
+        if value is not None:
```

Comment:
I'm not sure about this one. Maybe it's good to know that `None` is a possible value?

Reply:
Okay, let's remove it then!